helm-charts/k8s-mediaserver/values.yaml

```diff
@@ -279,3 +280,38 @@ plex:
   #  accessModes: ReadWriteOnce
   #  storage: 5Gi
   #  selector: {}
+
+jellyfin:
+  enabled: true
```

Comment:
We're enabling both plex and jellyfin by default? Seems like most users will use one or the other. I'd set it as `enabled: false` to give the option only.